src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/TelemetryHttpModule.cs

```diff
@@ -26,6 +28,16 @@ namespace OpenTelemetry.Instrumentation.AspNet
     /// </summary>
     public class TelemetryHttpModule : IHttpModule
     {
+        /// <summary>
+        /// OpenTelemetry.Instrumentation.AspNet <see cref="ActivitySource"/> name.
+        /// </summary>
+        public const string AspNetSourceName = "OpenTelemetry.Instrumentation.AspNet.Telemetry";
+
+        /// <summary>
+        /// <see cref="Activity.OperationName"/> for OpenTelemetry.Instrumentation.AspNet created <see cref="Activity"/> objects.
+        /// </summary>
+        public const string AspNetActivityName = "Microsoft.AspNet.HttpReqIn";
```

Comment:
Do we really need this to be a public const?
Public consts are actually "linked into" the target assembly, so if we change the value and do a binplace (rather than recompile), there will be a mismatch. The normally practice I've seen is that we only put something that are truly constants - for example `PI = 3.1415926`.

Reply:
They don't need to be public. The reason I did that was so our instrumentation can use them and we don't need a magic string. We really only need the ActivitySource name (not AspNetActivityName). What was interesting is that the public api analyzer picked them up. So I guess that is its way of saying: if we make them public they are part of the public api and we should never change them. Really changing them would be breaking, so I kind of like being part of the contract 🤷